pkg/synopsysctl/cmd_deploy.go

```diff
@@ -154,4 +154,9 @@ func init() {
 	deployCmd.Flags().BoolVar(&dryRun, "dryRun", dryRun, "dry run to run the test cases")
 	deployCmd.Flags().StringVarP(&logLevel, "log-level", "l", logLevel, "log level of synopsys operator")
 	deployCmd.Flags().IntVarP(&threadiness, "no-of-threads", "c", threadiness, "number of threads to process the custom resources")
+
+	// user is explicitly required to set the postgres passwords for: 'admin', 'postgres', and 'user'
+	cobra.MarkFlagRequired(deployCmd.Flags(), "admin-password")
```

Comment:
In the help text of all passwords, the default value is set to "blackduck" but in the above code, those fields marked for required, both are contradicting to each other.

Do we need to remove the default password?